src/util/Util.h

```diff
@@ -56,7 +56,12 @@ class Util
 
 	static gboolean paintBackgroundWhite(GtkWidget* widget, cairo_t* cr, void* unused);
 
+	static char* getCoordinateString(double xVal, double yVal);
+	
+	static char* getFormatdPrecisionString();
 };
 
 static const size_t size_t_npos = static_cast<size_t>(-1);
 // for 64b systems it's 18446744073709551615 and for 32b – 4294967295
+
+static const int PRECISION_DECIMAL_PLACES = 8;
```

Comment:
Intead of int, I suggest you create a 
`static const char* PRECISION_FORMAT_STRING;`
and initialize it in .cpp